aiida/tools/groups/__init__.py

```diff
@@ -0,0 +1,14 @@
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+# pylint: disable=wildcard-import,undefined-variable
+"""Provides tools for interacting with AiiDA Groups."""
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
```

Comment:
remove lines

Reply:
done in 1e9caab0217ba9eaa43153d7b47a56999d4dfebc